vm-virtio/src/vhost_user/mod.rs

```diff
@@ -0,0 +1,109 @@
+// Copyright 2019 Intel Corporation. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 AND BSD-3-Clause
+
+extern crate epoll;
+extern crate net_util;
+extern crate vhost_rs;
+extern crate virtio_bindings;
+extern crate vm_memory;
+
+use std;
+use std::io;
+use vhost_rs::vhost_user::Error as VhostUserError;
+use vhost_rs::Error as VhostError;
+use vm_memory::Error as MmapError;
+use vm_memory::GuestMemoryError;
+
+mod handler;
+pub mod net;
+pub mod vu_common_ctrl;
+
+pub use self::net::CtlVirtqueue;
+pub use self::net::Net;
+pub use self::vu_common_ctrl::VhostUserConfig;
+
+pub const VIRTIO_NET_CTRL_MQ_VQ_PAIRS_MIN: u16 = 1;
+pub const VIRTIO_NET_CTRL_MQ_VQ_PAIRS_MAX: u16 = 0x8000;
```

Comment:
We don't need this as they're already defined in the virtio-bindings crate.

Reply:
use virtio-bindings instead, also move this to new PR.